share/functions/export.fish

```diff
@@ -10,7 +10,7 @@ function export --description 'Set global variable. Alias for set -gx, made for
                 set -gx $v $$v
             case 2
                 if contains $v[1] PATH CDPATH MANPATH
-                    set -gx $v[1] (string split ":" $v[2])
+                    set -gx $v[1] (string split ":" (string replace " " ":" $v[2]))
```

Comment:
Just so you don't miss it (github shows it as a comment on outdated code): You want `contains -- $v[1]` to guard against a $v[1] starting with a "-".